### PR TITLE
Generate majority of indices operations

### PR DESCRIPTION
**Pull request by Xtansia**
### Description
Generates all but one indices operation that requires some extra work/handling that will be done separately.

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.
For more information on following Developer Certificate of Origin and signing off your commits, please check [here](https://github.com/opensearch-project/OpenSearch/blob/main/CONTRIBUTING.md#developer-certificate-of-origin).

### CODE REVIEW

File: java-client/src/generated/java/org/opensearch/client/opensearch/_types/GetStats.java [hunk 1]
```diff
@@ -193,61 +202,96 @@ protected void serializeInternal(JsonGenerator generator, JsonpMapper mapper) {
...
         generator.write(this.missingTimeInMillis);
 
         generator.writeKey("missing_total");
         generator.write(this.missingTotal);
 
         if (this.time != null) {
-            generator.writeKey("time");
-            generator.write(this.time);
-
+            generator.writeKey("getTime");
```
Comment (reta) [inline java-client/src/generated/java/org/opensearch/client/opensearch/_types/GetStats.java:215]: The `getTime` does not look right. shouldn't be `time`?:

```suggestion
            generator.writeKey("time");
```

Comment (Xtansia) [reply java-client/src/generated/java/org/opensearch/client/opensearch/_types/GetStats.java:215]: The field is actually `getTime`: https://github.com/opensearch-project/OpenSearch/blob/2.x/server/src/main/java/org/opensearch/index/get/GetStats.java#L158

Looks like this bug was introduced nearly twelve years ago: https://github.com/opensearch-project/OpenSearch/commit/cc83c2f848be69a77f1275fe1ff5363dcdd4c955#diff-f7abcc8f5eb3ec75bb9be11b7fe88baa0bcedf62c3c481552a0e98ceb7125a27R124

Comment (reta) [reply java-client/src/generated/java/org/opensearch/client/opensearch/_types/GetStats.java:215]: Thanks @Xtansia

Comment (reta) [reply java-client/src/generated/java/org/opensearch/client/opensearch/_types/GetStats.java:215]: Do you mind creating an issue on OpenSearch side?

**Review by reta — APPROVED** [review]

File: java-client/src/generated/java/org/opensearch/client/opensearch/_types/GetStats.java (hunk 1 above)
Comment (Xtansia) [reply java-client/src/generated/java/org/opensearch/client/opensearch/_types/GetStats.java:215]: Created https://github.com/opensearch-project/OpenSearch/issues/16894

**Pull request merged**